### ISSUE TITLE
Bisect result shows up as an error

### ISSUE THREAD

**Issue by Trzyipolkostkicukru** [ForkIssues/Tracker#1473]
When you finish bisection in Fork, it shows you an error alert instead of an info alert or a dedicated alert. The first time I saw it, I immediately closed it without reading the details, thinking "oh no, git is cranky today, I need to try again". 

Fork Version 2.11.3 on macos

![Zrzut ekranu 2021-09-21 o 10 35 15](https://user-images.githubusercontent.com/5207692/134139136-92896ffa-3c6b-4011-83b9-c3006ed9f5d8.png)

**Comment by gotoh** [comment]
I have same issue with ver. 1.74.1
![image](https://user-images.githubusercontent.com/105501/175021878-471f4620-2201-4632-a1f9-295977546de5.png)

**Comment by DanPristupov** [comment]
Yes, we need to create a separate bisect result windows instead of reusing the error dialog.

**Comment by kambala-decapitator** [comment]
also noticed that in 2.52.0
